zmk/backports.py

```diff
@@ -4,9 +4,9 @@
 
 # pyright: reportMissingImports = false
 
+# NOTE(elpekenin): aliased import prevents "unused import" error
+
 try:
-    # pylint: disable=unused-import
-    from enum import StrEnum
+    from enum import StrEnum as StrEnum
```

Comment:
This doesn't seem to prevent unused-import. It just gives me both unused-import and useless-import-alias instead.